NEW_README.md

```diff
@@ -141,15 +140,155 @@ Run prints its output if either,
 2. no tasks are "long" (eg a one-shot "build" procedure, rather than an ongoing
    "dev server").
 
+## Task Configuration Details
+
+The core of Run's configuration is the `tasks.toml` file, which defines the tasks you want Run to manage. Each task is specified with a range of properties, as shown in the example below:
+
+```toml
+[[task]]
+  id = "dev"
+  type = "long"
+  dependencies = ["simulate-coding"]
+  triggers = ["build-css", "build-js"]
+  watch = ["server-config.json"]
+  env = {
+    KEY = "some value"
+    OTHER_KEY = "some other value"
+  }
+  cmd = """
+    echo "dev-server running at http://localhost:3000"
+    while true; do sleep 1; done
+  """
+```
+
+Explore a [realistic project configuration](https://github.com/amonks/run/tree/main/examples/demo) to see `tasks.toml` in action.
+
+### Understanding Task Fields
```

Comment:
review note:
this is all new copy, loosely based on the existing "task files" section

Reply:
review:
this copy is a plain improvement over the existing "task files" section, and should be merged.